R/learner_flexsurv_surv_flexible.R

```diff
@@ -111,8 +111,8 @@ predict_flexsurvreg = function(object, task, learner, ...) {
   }
 
   X = stats::model.matrix(formulate(rhs = task$feature_names),
-    data = newdata,
-    xlev = task$levels())
+                          data = newdata,
```

Comment:
I am not too fond of this type of indentation style as it wastes a lot of space for nothing